mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -173,7 +173,7 @@ class BroadcastConverter final
       SmallVector<Attribute, 5> newShapeAttr;
 
       // align the dimensions - by the given axis
-      for (int i = 0; i < outRank; i++) {
+      for (uint i = 0; i < outRank; i++) {
```

Comment:
I'm pretty sure we usually spell that `unsigned` upstream (or perhaps `unsigned int`)

Reply:
Old habits.  I'll check.

Reply:
Should be uint32_t, to match the definition of outRank.